torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -1,77 +1,82 @@
-from __future__ import annotations
-
 import inspect
-
 import operator
-
 import re
 import types
 import warnings
-
 from typing import Callable, Dict, List, Optional, Sequence, Tuple, Union
 
 import onnxscript  # type: ignore[import]
-from onnxscript import evaluator, opset18  # type: ignore[import]
-from onnxscript.function_libs.torch_lib import graph_building  # type: ignore[import]
+from onnxscript.function_libs.torch_lib import (  # type: ignore[import]
+    graph_building as onnxscript_graph_builder,
```

Comment:
```suggestion
    graph_building as onnxscript_graph_building,
```

consider this to preserve the concept?